pip install for Python Druid API (#13938)
apache/druid: examples/quickstart/jupyter-notebooks/druidapi/README.md

environment which complements the UI-based Druid console. The primary use of `druidapi` at present
is to support the set of tutorial notebooks provided in the parent directory.

> `druidapi` works against any version of Druid. Operations that exploit newer features obviously work

Reviewer: ```suggestion
`druidapi` works against any version of Druid. Operations that make use of newer features obviously work
```

git clone git@github.com:apache/druid.git
```

`druidapi` is located in `examples/quickstart/jupyter-notebooks/druidapi/`.

> From this directory, install the package and its dependencies with pip using the following command:

Reviewer 2: Took me a moment to realize you had introduced another directory level. That is helpful: there is not a place to put the test code that I currently have sitting off to the side. Would be good to mention this structure to make it obvious that there there are two `druidapi` directories here.

Author: Updated

```
pip install .

> ```

Reviewer 2: Same note as above.

Author: This is referenced in the previous few lines:
> `druidapi` is located in `examples/quickstart/jupyter-notebooks/druidapi/`.
From this directory, install the package ...

Verify your installation by checking that the following commands run in Python without error:

```python
import druidapi

> druid = druidapi.jupyter_client('http://localhost:8888')

Reviewer 2: Only the `import` is needed, which will fail if `druidapi` is not on the Python path.

Author: Updated

```

## Getting started

To use `druidapi`, first import the library, then connect to your cluster by providing the URL to your Router instance. The way that is done differs a bit between consumers.

> ### From a tutorial Jupyter Notebook

Reviewer 2: Always odd that we don't use title case in titles... Is Notebook a proper noun here? Or, should this be:

> From a tutorial Jupyter notebook

Author: We use sentence case for headings as per the Google style guide: https://developers.google.com/style/capitalization. Will update `notebook` to lowercase n.

git clone git@github.com:apache/druid.git
```

`druidapi` is located in `examples/quickstart/jupyter-notebooks/druidapi/`.
From this directory, install the package and its dependencies with pip using the following command:

```
pip install .
```

Note that there is a second level `druidapi` directory that contains the modules. Do not run

> the install command in this internal folder.

Reviewer: We usually refer to it as a child or subdirectory and not an internal folder, right?

Author: ```suggestion
the install command in the subdirectory.
```

> Verify your installation by checking that the following command runs in Python without error:

Reviewer: ```suggestion
Verify your installation by checking that the following command runs in Python:
```

```python
import druidapi
```

> ## Getting started

Reviewer: ```suggestion
The import statement should not return anything if it runs successfully.

## Getting started
```
